File: src/main/java/site/gachontable/gachontablebe/global/jwt/exception/UnsupportedTokenException.java
package site.gachontable.gachontablebe.global.jwt.exception;

import site.gachontable.gachontablebe.global.error.ErrorCode;
import site.gachontable.gachontablebe.global.error.exception.ServiceException;

public class UnsupportedTokenException extends ServiceException {
    public UnsupportedTokenException() {
        super(ErrorCode.UNSUPPORTED_TOKEN);
    }
}

File: src/main/java/site/gachontable/infra/security/jwt/exception/UnsupportedTokenException.kt
package site.gachontable.infra.security.jwt.exception;

import site.gachontable.independent.type.ErrorCode;
import site.gachontable.independent.error.exception.ServiceException;

public class UnsupportedTokenException extends ServiceException {
    public UnsupportedTokenException() {
        super(ErrorCode.UNSUPPORTED_TOKEN);
    }
}
